src/modm/platform/adc/stm32/adc_impl.hpp.in

```diff
@@ -257,3 +257,57 @@ modm::platform::Adc{{ id }}::acknowledgeInterruptFlags(const InterruptFlag_t fla
 {
 	ADC{{ per }}->SR = ~flags.value;
 }
+
+uintptr_t
+modm::platform::Adc{{ id }}::getDataRegisterPointer()
```

Comment:
Why needed? This looks a bit like a hack.

Reply:
we need it to pass the peripheral address to the DMA.